test/test_gnmi_appldb.py

```diff
@@ -5,410 +5,61 @@
-                'qos_02': {'bw': '6000', 'cps': '200', 'flows': '101'}
-            }
-        },
-        {
-            'update_path': '/sonic-db:APPL_DB/DASH_VNET',
-            'get_path': '/sonic-db:APPL_DB/_DASH_VNET',
-            'value': {
-                'Vnet3721': {
-                    'address_spaces': ["10.250.0.0", "192.168.3.0", "139.66.72.9"]
-                }
-            }
-        }
-    ],
-    [
-        {
-            'update_path': '/sonic-db:APPL_DB/DASH_QOS/qos_01',
-            'get_path': '/sonic-db:APPL_DB/_DASH_QOS/qos_01',
-            'value': {'bw': '10001', 'cps': '1001', 'flows': '101'}
-        },
-        {
-            'update_path': '/sonic-db:APPL_DB/DASH_QOS/qos_02',
-            'get_path': '/sonic-db:APPL_DB/_DASH_QOS/qos_02',
-            'value': {'bw': '10002', 'cps': '1002', 'flows': '102'}
-        },
-        {
-            'update_path': '/sonic-db:APPL_DB/DASH_VNET/Vnet3721',
-            'get_path': '/sonic-db:APPL_DB/_DASH_VNET/Vnet3721',
-            'value': {
-                'address_spaces': ["10.250.0.0", "192.168.3.0", "139.66.72.9"]
-            }
-        }
-    ]
-]
-
-def clear_appl_db(table_name):
-    prefix = '/sonic-db:APPL_DB'
-    get_path = prefix + '/_' + table_name
-    ret, msg_list = gnmi_get([get_path])
-    if ret != 0:
-        return
-    for msg in msg_list:
-        rx_data = json.loads(msg)
-        delete_list = []
-        for key in rx_data.keys():
-            delete_path = prefix + '/' + table_name + '/' + key
-            delete_list.append(delete_path)
-        if len(delete_list):
-            ret, msg = gnmi_set(delete_list, [], [])
-            assert ret == 0, msg
-
 class TestGNMIApplDb:
 
-    @pytest.mark.parametrize('test_data', test_data_update_normal)
-    def test_gnmi_update_normal_01(self, test_data):
-        clear_appl_db('DASH_QOS')
-        clear_appl_db('DASH_VNET')
+    def test_gnmi_update_normal_01(self):
+        proto_bytes = b"\n\x010\x12$b6d54023-5d24-47de-ae94-8afe693dd1fc\x1a\x17\n\x12\x12\x10\r\xc0-\xdd\x82\xa3\x88;\x0fP\x84<\xaakc\x16\x10\x80\x01\x1a\x17\n\x12\x12\x10-\x0e\xf2\x7f\n~c_\xd8\xb7\x10\x84\x81\xd6'|\x10\x80\x01\x1a\x17\n\x12\x12\x10\x1bV\x89\xc8JW\x06\xfb\xad\b*fN\x9e(\x17\x10\x80\x01\x1a\x17\n\x12\x12\x107\xf9\xbc\xc0\x8d!s\xccVT\x88\x00\xf8\x9c\xce\x90\x10\x80\x01\x1a\x17\n\x12\x12\x10\tEb\x11Mf]\x12\x17x\x99\x80\xea\xd1u\xb4\x10\x80\x01\x1a\x17\n\x12\x12\x10\x1f\xd3\x1c\x89\x99\x16\xe7\x18\x91^0\x81\xb1\x04\x8c\x1e\x10\x80\x01\x1a\x17\n\x12\x12\x10\x06\x9e55\xdb\xb5&\x93\x99\xfaC\x81\x16P\xdc\x1d\x10\x80\x01\x1a\x17\n\x12\x12\x10&]U\x96e4\xf4\xd2'&\x04i\xdf\x8dA\x9f\x10\x80\x01\x1a\x17\n\x12\x12\x108\xd5\xa3*\xe7\x80\xdc\x1e\x80f\x94\xb7\xb6\x86~\xcd\x10\x80\x01\x1a\x17\n\x12\x12\x101\xf0@F\nu+}\x1e\"\\\\\xdb\x01\xe3\x82\x10\x80\x01\"\x05vnet1\"\x05vnet2\"\x05vnet1\"\x05vnet2\"\x05vnet2\"\x05vnet1\"\x05vnet2\"\x05vnet2\"\x05vnet1\"\x05vnet1"
```

Comment:
Is there are util to convert json format data to protocol buffer format?

Reply:
We will not provide json format, and we need Ze to generate protobuf builder